src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -203,6 +211,16 @@ class JibriSelenium(
 
     fun leaveCallAndQuitBrowser() {
         emptyCallTask?.cancel(true)
+        File("/tmp/browser.out").printWriter().use { out ->
+            chromeDriver.manage().logs().availableLogTypes.forEach { logType ->
+                val logEntries = chromeDriver.manage().logs().get(logType)
+                logger.info("Got ${logEntries.all.size} log entries")
+                out.println("========= TYPE=$logType ===========")
+                logEntries.all.forEach {
+                    out.println(it.toJson())
+                }
+            }
+        }
```

Comment:
Is the additional logging left in on purpose? If so, do you really only want LogType.DRIVER logs?

Reply:
So I was going to yank this out before pushing this, but then thought they would be useful to have.  Then I was thinking about it more this morning and realized in order for them to be actually useful, they should write to more proper log files with log rotation and all that (otherwise they get overwritten every call).  So I'll decide whether to yank this for now or make it more official.

Reply:
Also, I found DRIVER to have the most useful information (for whatever reason).